content/guides/hanami/v2.3/assets/using-a-cdn.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Using A Cdn
```

Comment:
I have a general question about casing of heading throughout guides for 2.3. In current guides for 2.2, we use sentence case for headings ("[Building a web app](https://hanakai.org/guides/hanami/v2.2/getting-started/building-a-web-app)", "Working with dependencies" etc.) but here for 2.3 we have title case, and even a pretty extreme one with capitalized articles and conjunctions. Is this an intentional change?